src/main/java/org/cyclops/integrateddynamics/part/aspect/Aspects.java

```diff
@@ -538,22 +547,27 @@ public static void load() {}
             public static final IAspectRead<ValueTypeBoolean.ValueBoolean, ValueTypeBoolean> BOOLEAN_LOW =
                     AspectReadBuilders.Redstone.BUILDER_BOOLEAN.handle(
                         input -> input == 0
-                    ).handle(AspectReadBuilders.PROP_GET_BOOLEAN, "low").buildRead();
+                    ).withUpdateType(AspectUpdateType.BLOCK_UPDATE)
+                            .handle(AspectReadBuilders.PROP_GET_BOOLEAN, "low").buildRead();
             public static final IAspectRead<ValueTypeBoolean.ValueBoolean, ValueTypeBoolean> BOOLEAN_NONLOW =
                     AspectReadBuilders.Redstone.BUILDER_BOOLEAN.handle(
                         input -> input > 0
-                    ).handle(AspectReadBuilders.PROP_GET_BOOLEAN, "nonlow").buildRead();
+                    ).withUpdateType(AspectUpdateType.BLOCK_UPDATE)
+                            .handle(AspectReadBuilders.PROP_GET_BOOLEAN, "nonlow").buildRead();
             public static final IAspectRead<ValueTypeBoolean.ValueBoolean, ValueTypeBoolean> BOOLEAN_HIGH =
                     AspectReadBuilders.Redstone.BUILDER_BOOLEAN.handle(
                         input -> input == 15
-                    ).handle(AspectReadBuilders.PROP_GET_BOOLEAN, "high").buildRead();
+                    ).withUpdateType(AspectUpdateType.BLOCK_UPDATE)
+                            .handle(AspectReadBuilders.PROP_GET_BOOLEAN, "high").buildRead();
             public static final IAspectRead<ValueTypeBoolean.ValueBoolean, ValueTypeBoolean> BOOLEAN_CLOCK =
                     AspectReadBuilders.Redstone.BUILDER_BOOLEAN_CLOCK.handle(AspectReadBuilders.PROP_GET_BOOLEAN, "clock").buildRead();
 
             public static final IAspectRead<ValueTypeInteger.ValueInteger, ValueTypeInteger> INTEGER_VALUE =
-                    AspectReadBuilders.Redstone.BUILDER_INTEGER.handle(AspectReadBuilders.PROP_GET_INTEGER, "value").buildRead();
+                    AspectReadBuilders.Redstone.BUILDER_INTEGER.withUpdateType(AspectUpdateType.BLOCK_UPDATE)
+                            .handle(AspectReadBuilders.PROP_GET_INTEGER, "value").buildRead();
             public static final IAspectRead<ValueTypeInteger.ValueInteger, ValueTypeInteger> INTEGER_COMPARATOR =
-                    AspectReadBuilders.Redstone.BUILDER_INTEGER_COMPARATOR.handle(AspectReadBuilders.PROP_GET_INTEGER, "comparator").buildRead();
+                    AspectReadBuilders.Redstone.BUILDER_INTEGER_COMPARATOR.withUpdateType(AspectUpdateType.BLOCK_UPDATE)
```

Comment:
Can't this also change without a block update?

Reply:
It should only change with neighbour changes, see the code in `TileEntity#markDirty`.